.github/workflows/createtagandrelease.yml

```diff
@@ -0,0 +1,29 @@
+name: create tag and relese
+
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    if: github.event.pull_request.merged && startsWith(github.head_ref, 'PR') && startsWith(github.base_ref, 'master')
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
+        id: tag_and_prepare_release
+        uses: moble/github-tag-action@main
+        env:
+          GITHUB_TOKEN:  ${{ secrets.CD_TOKEN }}
+          WITH_V: true
+          CUSTOM_TAG: "v7.2.0"
```

Comment:
custom tag should be dynamic right

Reply:
custom tag supposed to get change during release workflow